wire-ios-sync-engine/Source/Calling/WireCallCenterV3.swift

```diff
@@ -963,7 +963,7 @@ private extension ZMConversation {
         case (.group, .proteus):
             return .conference
 
-        case (.group, .mls):
+        case (.group, .mls), (.`self`, .mls):
```

Comment:
@johnxnguyen @David-Henner is it safe to assume in all cases: messages from self conv mls will result with this type?

Reply:
I think for now yes, but probably not if we support MLS 1:1 calls.

Reply:
Why do we need to consider the mls self group as an mls conference?

Reply:
otherwise the "reject" message does not go through the workflow, it requires an avsconversationType. It seemed the most appropriate one